client/web/dev/utils/csrf/get-csrf-token-cookie-middleware.ts

```diff
@@ -4,6 +4,7 @@ import { CSRF_COOKIE_NAME } from './get-csrf-token-and-cookie'
 
 // Attach `CSRF_COOKIE_NAME` cookie to every response to avoid "CSRF token is invalid" API error.
 export const getCSRFTokenCookieMiddleware = (csrfCookieValue: string): RequestHandler => (_request, response, next) => {
+    _request.headers['x-requested-with'] = 'Sourcegraph FE local server'
```

Comment:
```suggestion
export const getCSRFTokenCookieMiddleware = (csrfCookieValue: string): RequestHandler => (request, response, next) => {
    request.headers['x-requested-with'] = 'Sourcegraph FE local server'
```

I'm assuming the variable was previously prefixed with `_` as it was unused